fv/connectivity/conncheck.go

```diff
@@ -460,6 +475,11 @@ func (e Expectation) Matches(response *Result, checkSNAT bool) bool {
 
 	} else {
 		if response != nil {
```

Comment:
I think the response can now equal nil, but we want the check to fail if e.Error is set but the response is nil.